app/lib/config.js

```diff
@@ -1,9 +1,17 @@
+function getRule() {
+  if (process.env.DISABLE_SCHEDULER === 'true') {
+    return new Date(2117, 11, 31, 23, 59);
```

Comment:
Why this date? Could it be less special looking e.g. `new Date(2100, 00, 00, 00, 00)`?